pebblo/app/service/local_ui/loader_apps.py

```diff
@@ -206,15 +206,21 @@ def get_all_loader_apps(self):
             _, ai_loader_apps = self.db.query(table_obj=AiDataLoaderTable)
 
             # Preparing all loader apps
+            app_processed = list()
             all_loader_apps: list = []
             for loader_app in ai_loader_apps:
                 app_data = loader_app.data
                 if app_data.get("docEntities") not in [None, {}] or app_data.get(
```

Comment:
we can just do  if app_data.get("docEntities")  or app_data.get("docTopics")

Reply:
This is to skip processing even if add_data = {"docEntities": None} or app_data = {"docEntities": {}} or app_data = {}

Reply:
Fixed.